.github/workflows/spam-comment-detection.yaml

```diff
@@ -0,0 +1,45 @@
+name: Spam detection
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  detect-spam:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@main
+
+      - name: Check for Spam
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const comment = process.env.COMMENT_BODY
+            const spam_words = ['Neuron Support Request']
```

Comment:
Could we add this variable to GitHub action env?

Reply:
Update by loading spam word list from the secrets

https://github.com/nervosnetwork/neuron/pull/3288/files#diff-c369c155176306e9be1a7df1156a1408083a4b28c9bc392758336e0ec9ff8e05R18